include/alpaka/kernel/TaskKernelCpuOmp2Threads.hpp

```diff
@@ -171,21 +173,13 @@ namespace alpaka
                                     throw std::runtime_error("The OpenMP 2.0 runtime did not create a parallel region!");
                                 }
 
-                                // GCC fails with:
-                                // error: redeclaration of const int& iBlockThreadCount
-                                // if(numThreads != iBlockThreadCount)
-                                //                  ^
-                                // note: const int& iBlockThreadCount previously declared here
-                                // #pragma omp parallel num_threads(iBlockThreadCount)
-                                //         ^
-#if (!BOOST_COMP_GNUC) || (BOOST_COMP_GNUC >= BOOST_VERSION_NUMBER(6, 0, 0))
```

Comment:
Note: this condition is superseded by the new one, so became useless and I removed it.